modules/nf-core/rtgtools/format/main.nf

```diff
@@ -0,0 +1,66 @@
+process RTGTOOLS_FORMAT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::rtg-tools=3.12.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/rtg-tools:3.12.1--hdfd78af_0':
+        'quay.io/biocontainers/rtg-tools:3.12.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(input1), path(input2), path(sam_rg)
+
+    output:
+    tuple val(meta), path("*.sdf"), emit: sdf
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def single = meta.containsKey("single_end") ? meta.single_end : true
+
+    def input = single ? "${input1}" : "--left ${input1} --right ${input2}"
+    def rg = sam_rg ? "--sam-rg ${sam_rg}" : ""
```

Comment:
can this go in  `ext.args`?

Reply:
It can be both, this argument asks for a file containing the read group or a string (see the meta file)